### ISSUE TITLE
Detect simple equality expression subjects with isTrue & isFalse assertions

### ISSUE THREAD

**Issue by jzbrooks** [jzbrooks/assertk-lint#48]
```kotlin
assertThat(name == "Justin").isTrue()
```

This type of assertion is common among new fluent assertion users and should be easy to detect and carry a high pedagogical value.

**Comment by jzbrooks** [comment]
This needs to be handled in a way that cooperates well with #43. One approach would be to re-cast that check as only being concerned with `assertThat(obj).isEqualTo(null)` and the like, then make _this_ check concerned with _all_ simple binary operations.

Which would result in 

```kotlin
assertThat(obj == null).isTrue()
```

being quick fixed by this check to 

```kotlin
assertThat(obj).isEqualTo(null)
```

and finally another quick fix by #43's detector

```
assertThat(obj).isNull()
```

I don't _love_ having to execute two quick fixes to make something like that happen, but that seems preferable to two detectors with potential overlap. Other solutions probably exist.

**Comment by jzbrooks** [comment]
A better solution is probably to have dedicated detectors and ensure in the implementations that they don't overlap. e.g. For `==` + `isTrue` detection, ensure the comparison isn't with null. It avoids the double quick fix scenario.
